src/components/error/ComponentErrorOverlay.react.js

```diff
@@ -1,66 +1,29 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-
-
-const styles = {
-  root: {
-    position: 'relative'
-  },
-  overlay: {
-    position: 'relative',
-    padding: '2px',
-    backgroundColor: 'rgb(255, 0, 0, .5)',
-    border: '1px solid black',
-    borderRadius: '2px',
-    display: 'inline-block'
-  },
-  childWrapper: {
-    position: 'relative',
-    zIndex: -1
-  },
-  popOverOpen: {'display': 'inline'},
-  popOverClosed: {'display': 'none'}
-}
+import './ComponentErrorOverlay.css';
 
 export default class ComponentErrorOverlay extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      popoverOpen: false
+    constructor(props) {
+        super(props);
     }
-    this.togglePopOver = this.togglePopOver.bind(this);
-  }
 
-  togglePopOver() {
-    this.setState({
-      popoverOpen: !this.state.popoverOpen
-    });
-  }
+    render() {
+        const {
+            componentId,
+            componentType,
+        } = this.props;
+        const errorLocationString =
+            'Error in ' + componentType + '(id=' + componentId + ')';
+        // const errorString = error.name + ' -- ' + error.message;
 
-  render() {
-    const { error, componentId, componentType, resolve, children } = this.props;
-    const errorLocationString = "Error in " + componentType + '(id=' + componentId + ')';
-    const errorString = error.name + " -- " + error.message;
-    return (
-      <div style={styles.root}>
-        <div
-          onClick={this.togglePopOver}
-          style={styles.overlay}>
-          <div style={styles.childWrapper}>
-            {children}
-          </div>
-          <div
-            style={this.state.popoverOpen ? styles.popOverOpen : styles.popOverClosed}
-            toggle={this.togglePopOver}
-          >
-            <strong>{errorLocationString}</strong>
-            <p>{errorString}</p>
-            <button onClick={resolve}>Resolve Error</button>
-          </div>
-        </div>
-      </div>
-    )
-  }
+        return (
+            <div
+                className={'dash-error-overlay'}
```

Comment:
That's a good point - I removed the underscore from those prefixes, because in other places such as dash-core-components we don't use them either. Basically it would be more consistent to not use underscores, as that's what other Dash repo's use. `_dash-loading` will also disappear once the Loading component and Loading States are merged. If it's a big issue for you, I can switch everything back to use `_dash`, otherwise I suggest we keep it as-is.